Dangerfile.swift

```diff
@@ -0,0 +1,12 @@
+import Danger
+
+let danger = Danger()
+
+let allSourceFiles = danger.git.modifiedFiles + danger.git.createdFiles
+
+let changelogChanged = allSourceFiles.contains("CHANGELOG.md")
```

Comment:
I recommend we add comments before each rule in this file. If more rules get added then it will make telling one rule a part from another easier. It also helps understand what the rule does and why it's there.

Reply:
Might be nice to host this separately as well on the BRS GitHub so we can (eventually) move it to other projects with a consistent set up.

Reply:
That's a good suggestion. In the future, we might even want to break things up into different methods, but I just added a comment for now.

Reply:
@wmcginty definitely! Danger-swift has support for plugins so I think it would just be a matter of developing our own, hosting it, and then implementing a standardized Dangerfile.swift across all of our repos. The main blocker to doing that right now is just getting a BR-owned GitHub bot account that Danger can use to post comments on PRs. Right now it's using my personal bot account for testing, but I'll see if I can get the BR account created soon.